README.md

```diff
@@ -18,7 +18,7 @@ Current notebook solutions, like Jupyter and Zeppelin, are lacking in some funda
   you're writing. TeX equations are also supported.
 - *Multi-language support* – Polynote allows you to mix multiple languages in one notebook, while sharing definitions
   seamlessly between them.
-- *Runtime insight* – Polynote tries to keep you informed of what's going on at runtime:
+- *Runtime insight* – Polynote keeps you informed of what's going on at runtime:
```

Comment:
TBH I don't think the docs really ought to project confidence at this stage. It's certainly not guaranteed that Polynote will be able to keep you informed of everything that's going on 😄 (hence the original wording).

I appreciate you taking the time to help out with the docs, but I'm inclined to stick with the original phrasing for this particular thing (at least for the time being).